commands/command_migrate.go

```diff
@@ -151,17 +151,24 @@ func includeExcludeRefs(l *tasklog.Logger, args []string) (include, exclude []st
 			include = append(include, ref.Refspec())
 		}
 	} else {
-		// Otherwise, if neither --include-ref=<ref> or
-		// --exclude-ref=<ref> were given, include no additional
-		// references, and exclude all remote references that are remote
-		// branches or remote tags.
-		remoteRefs, err := getRemoteRefs(l)
+		bare, err := git.IsBare()
 		if err != nil {
-			return nil, nil, err
+			return nil, nil, errors.Wrap(err, "fatal: unable to determine bareness")
```

Comment:
I think this error wrapping is unnecessary, as any error will be incredibly obvious. The error from `getRemoteRefs()` isn't wrapped.

Reply:
Hm, I think that keeping the error wrapping might make sense, since `git rev-parse --is-bare-repository` can fail in weird ways, like:

```
~/not-a-real-dir $ git rev-parse --is-bare-repository
fatal: Unable to read current working directory: No such file or directory
```

Reply:
How do you get that though? If the `git` binary is missing?

Reply:
This one's tricky -- I am able to reproduce it when the current directory doesn't exist, i.e., if you change branches from B -> A where B has directory D but A does not.